evm/vm/flavors/mainnet/__init__.py

```diff
@@ -8,7 +8,7 @@
 )
 
 
-MainnetEVM = EVM.configure(
+MainnetEVM = Chain.configure(
```

Comment:
Ah, I see what you're getting at here.  Yes, `MainnetEVM` should now be `MainnetChain`.  I'm good fixing this in a subsequent cleanup PR.